scripts/api/data/licenses/licenseCC-BY-4.0.json

```diff
@@ -1,8 +1,11 @@
 {
-  "name": "CC BY 4.0",
-  "uri": "http://creativecommons.org/licenses/by/4.0",
-  "shortDescription": "Creative Commons Attribution 4.0 International License.",
-  "iconUrl": "https://licensebuttons.net/l/by/4.0/88x31.png",
-  "active": true,
+  "rightsName": "CC BY 4.0",
+  "rightsURI": "https://creativecommons.org/licenses/by/4.0/",
+  "rightsIdentifier": "CC-BY-4.0",
+  "rightsIdentifierScheme": "SPDX",
+  "schemeURI": "https://spdx.org/licenses/",
+  "rightsShortDescription": "Creative Commons Attribution 4.0 International.",
+  "rightsIconUrl": "https://licensebuttons.net/l/by/4.0/88x31.png",
+  "rightsActive": "true",
```

Comment:
Not surprisingly `edu.harvard.iq.dataverse.api.Licenses.addLicense` is failing because the field names have changed. In this stack trace, for example, the code is looking for `uri` rather than `rightsURI`: [addlicense.txt](https://github.com/IQSS/dataverse/files/10352695/addlicense.txt)

This pull request is great for showing a future direction, but the code needs to be adjust to add new fields to the database and we'd need to write SQL upgrade scripts to migrate the old licenses into the new fields.